tests/neo4j/datatypes/_util.py

```diff
@@ -0,0 +1,607 @@
+TZ_IDS = (
```

Comment:
how are we going to maintain this over time?

Reply:
I don't quite know. Any suggestions welcome. I mean this list should probably only ever get extended and never reduced.